permissions/plugin-dotnet-as-script.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: "dotnet-as-script"
+paths:
+- "org/jenkins-ci/plugins/dotnet-as-script"
```

Comment:
Wrong path due to https://github.com/jenkinsci/dotnet-as-script-plugin/blob/805a9e36bf77cdeb32648d9c98f45a39b31278f7/pom.xml#L12

Please see this repo's readme how to map from groupId/artifactId to path.